software/earl/http-api.go

```diff
@@ -112,6 +141,9 @@ func (event *JsonAppEvent) writeJSONEvent(out http.ResponseWriter, jsonp_callbac
 }
 
 func (a *ApiServer) ServeHTTP(out http.ResponseWriter, req *http.Request) {
+	begin := time.Now()
+	defer httpRequestDurationSeconds.With(prometheus.Labels{"method":req.Method}).Observe(time.Since(begin).Seconds())
```

Comment:
Isn't seconds a little bit coarse-grained ? I'd probably choose microseconds as unit.

Reply:
In Prometheus, we always use the base units. All Prometheus values are float64. Go [time.Since()](https://golang.org/pkg/time/#Since) is a duration in nanoseconds, so we won't lose any precision here. :grin: